best_practices/forms.rst

```diff
@@ -51,7 +52,8 @@ form in its own PHP class::
 
 To use the class, use ``createForm`` and instantiate the new class::
 
-    use AppBundle\Form\PostType;
+    // src/AppBundle/Controller/PostController.php
```

Comment:
I don't think we need this here (it does not really matter how your controller is named and we actually omit most of the class already for readability). Though I would now move the `use` statement below the placeholder comment.